### PR TITLE
add swell lrt

### PR DESCRIPTION
**Pull request by Zishan-7**
- Added swell to LRT chart.

### CODE REVIEW

File: src/lrt/LRTDistribution.jsx [hunk 1]
```diff
@@ -341,22 +341,22 @@ export default function LRTDistribution({ data, height }) {
...
           <ul className="text-sm">
             {Object.keys(tooltipData.data.protocols)
               .sort()
               .map(key => (
                 <li key={`tt-${key}`}>
-                  <div className="flex flex-row gap-1 items-center">
+                  <div className="flex flex-row items-center gap-1">
                     <span
-                      className="h-3 inline-block rounded-full w-3"
+                      className="inline-block w-3 h-3 rounded-full"
```
Comment (rubo) [inline src/lrt/LRTDistribution.jsx:359]: The changes to this line seem unnecessary. Please revert. Below is alright.

**Review by rubo — CHANGES_REQUESTED** [review]

File: src/lrt/LRTDistribution.jsx (hunk 1 above)
Comment (Zishan-7) [reply src/lrt/LRTDistribution.jsx]: Done 👍

Comment (rubo) [reply src/lrt/LRTDistribution.jsx]: I mean all the changes up to this line :)

Comment (Zishan-7) [reply src/lrt/LRTDistribution.jsx]: Ah, sorry for the confusion, I have fixed it now.

**Review by rubo — APPROVED** [review]

**Pull request merged**